lambdaclass/game_backend
# Add flame_on

## Changes:
- Add flamegraphs from the phoenix dashboard.

## How to test:
1. Set the env variable:
   ```bash 
      export FLAMEGRAPH="true"
   ```
2. Recompile the project:
   ```bash 
    mix compile --force
   ```
3. Start the server:
  ```bash 
   iex -S mix phx.server
   ```
4. Go to: 
```
http://localhost:4000/dev/dashboard/flame_on
```
5. Choose a module + function with arity and start a game or load test or even call it from the elixir repl, you should see something like this:
![image (2)](https://github.com/lambdaclass/game_backend/assets/56402156/96f82520-d845-4d6e-83f7-cc1a3132d23c)
